includes/api/save-abandonment-data.php

```diff
@@ -55,6 +56,14 @@ function saveCartAbandonmentData(WP_REST_Request $request)
     rzpLogInfo('Woocommerce order id:');
     rzpLogInfo(json_encode($wcOrderId));
 
+    // Check Wati.io retargetting plugin is active or not
+    if (is_plugin_active('wati-chat-and-notification/wati-chat-and-notification.php')){
```

Comment:
@angbpy As it's entirely dependent on the customer's phone number, what is the phone number not yet entered?